cli/src/commands/mock.ts

```diff
@@ -50,7 +50,7 @@ export default class Mock extends Command {
       await runMockServer(contract, {
         port,
         pathPrefix: pathPrefix ?? "",
-        ...proxyConfig,
+        proxyConfig,
```

Comment:
This was a bug. The options passed to `runMockServer` expected a `ProxyConfig` object, but this call was exploding the proxy configuration instead of just passing it. This resulted in the proxy configuration never being used.